internal/organizations/organizations.go

```diff
@@ -68,6 +69,9 @@ func Create(ctx context.Context, kubeClient *kubernetes.Cluster, gitea GiteaInte
 		},
 		metav1.CreateOptions{},
 	); err != nil {
+		if apierrors.IsAlreadyExists(err) {
+			return errors.Errorf("Org '%s' name cannot be used. Please try another name", org)
+		}
```

Comment:
We do not have an acceptance test covering this code path, don't we ?

Reply:
Added a test.